tests/unit/cunit_test_main.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#define CATCH_CONFIG_RUNNER
+#include "mfem.hpp"
+#include "catch.hpp"
+
+int main(int argc, char *argv[])
+{
+   mfem::Device device("cuda");
```

Comment:
It might be useful to dump the device: I'm trying with the `debug` one, which shows a couple of errors.
I'm trying to see where do they come from.

Reply:
`Testing 3D partial assembly H(curl) diagonal for integrator 1 and coeffType 0: 1 elements.` has issues with `valgrind`.
On `lassen`, `MFEM_DEVICE=cpu valgrind ./cunit_tests` reproduces the error.
You can also remove the `#ifdef` in :
```
   //#ifdef MFEM_USE_CUDA
   cfg.testsOrTags.push_back("[CUDA]");
   //#endif
```

Reply:
The ifdef is now removed.